pkg/bot/slack_msg_tracker.go

```diff
@@ -0,0 +1,80 @@
+package bot
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/slack-go/slack"
+)
+
+const (
+	msgReceivedEmoji  = "eyes"
+	msgProcessedEmoji = "heavy_check_mark"
```

Comment:
I would use following to show completed process
![Screenshot 2023-07-20 at 10 59 35](https://github.com/kubeshop/botkube/assets/1237982/f6de8d7f-3eac-4be7-ba2f-1f94cd2649ac)
If you are using dark mode, marked icon is almost invisible as follows
![Screenshot 2023-07-20 at 11 00 36](https://github.com/kubeshop/botkube/assets/1237982/a81190fb-2120-4983-b076-2cf576a7dcc9)

Reply:
changed 👍 I also double-checked with Donna  and she also prefers "✅ "